CSS/CSS.sublime-syntax

```diff
@@ -271,6 +278,36 @@ contexts:
         - match: '\s*(?=[{;])'
           pop: true
         - include: media-query
+# Namespace At-Rule: https://www.w3.org/TR/css3-namespace/
+  namespace:
+    - match: \s*((@)namespace)\s*\b(\S*)
+      captures:
+        1: keyword.control.at-rule.namespace.css
+        2: punctuation.definition.keyword.css
+        3: entity.other.namespace-prefix.css
```

Comment:
I'd say let's skip adding the `punctuation.definition.keyword.css` for the `@` symbol here and on line 301. IMO, the `@` is really part of the keyword and not so much separate punctuation.